Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2893,7 +2893,6 @@ - (void)yapDatabaseModified:(NSNotification *)notification
                 case YapDatabaseViewChangeDelete: {
                     DDLogVerbose(@"YapDatabaseViewChangeDelete: %@, %@", rowChange.collectionKey, rowChange.indexPath);
                     [self.collectionView deleteItemsAtIndexPaths:@[ rowChange.indexPath ]];
-                    [rowsThatChangedSize removeObject:@(rowChange.indexPath.row)];
```

Comment:
The values in `rowsThatChangedSize` are expressed as a "new index path", not an "old index path".  By definition, deletes only affect old rows.

Reply:
So during a delete, we were failing to reload a cell. Specifically - they one found at the deleted cells old index - likely the subsequent interaction. 

This could account for message bubbles following a deleted interaction being mis-sized, no?

Reply:
That's about right.